arch/xtensa/Kconfig

```diff
@@ -81,6 +81,21 @@ config XTENSA_CP_INITSET
 		is provided by CONFIG_XTENSA_CP_INITSET.  Each bit corresponds to one
 		coprocessor with the same bit layout as for the CPENABLE register.
 
+config XTENSA_USE_SEPERATE_IMEM
+	bool "Use a seperate heap for internal memory"
+	default n
+
+config XTENSA_IMEM_REGION_SIZE
+	hex "DRAM region size for internal use"
+	depends on XTENSA_USE_SEPERATE_IMEM
+	range 0x2000 0x28000
+	default 0x28000
+
+config XTENSA_IMEM_PROCFS
```

Comment:
i feel it's more convenient for users to have this info in /proc/meminfo than the dedicated file. how do you think?

Reply:
I was concerned that this change would be too "invasive" in procfsmeminfo.  This feature is "too" Xtensa/ESP32 specific.